lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -159,4 +181,33 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
       {session.notebook_name, -DateTime.to_unix(session.created_at)}
     end)
   end
+
+  defp sort_sessions(sessions, "memory") do
+    sessions
+    |> Enum.map(&standardize/1)
```

Comment:
You can define `total_runtime_memory(session)` that returns total or 0, this should simplify this function and the one below too!